example/generated-src/objc/TXSItemList.mm

```diff
@@ -21,7 +21,14 @@ + (nonnull instancetype)itemListWithItems:(nonnull NSArray<NSString *> *)items
 
 - (NSString *)description
 {
-    return [NSString stringWithFormat:@"<%@ %p items:%@>", self.class, self, self.items];
+    return [NSString stringWithFormat:@"<class, %@ : %p, dict, %@>", self.class, self, [[self toDict] description]];
```

Comment:
I'm confused about the use of comma and colon here as separators, and find this format hard to understand.  I suggest going back to the old format, or maybe using a name=value or name:value layout more consistently across the 3 values you're providing.